core/tests/protractor_utils/general.js

```diff
@@ -256,6 +256,52 @@ var navigateToTopicsAndSkillsDashboardPage = async function() {
   await waitFor.pageToFullyLoad();
 };
 
+var goOffline = async function() {
+  await browser.driver.setNetworkConditions(
+    {
+      offline: true,
+      latency: 0,
+      download_throughput: 0,
+      upload_throughput: 0
+    });
+};
+
+var goOnline = async function() {
+  // Download throughput refers to the maximum number of bytes that can be
+  // downloaded in a given time.
+  // Upload throughput refers to the maximum number of bytes that can be
+  // uploaded in a given time.
+  await browser.driver.setNetworkConditions(
+    {
+      offline: false,
+      latency: 150,
+      download_throughput: 450 * 1024,
+      upload_throughput: 150 * 1024
+    });
+};
+
+var offlineAlert = async function() {
```

Comment:
Do these alerts appear on any other pages? If not, then they should go in the page-specific utility file, not `general`

Reply:
Functions should be named as verbs instead of nouns. For example `waitForOfflineAlert`

Reply:
Done.